frontend_app_folder_advanced/src/container/FolderAdvanced.jsx

```diff
@@ -81,15 +82,17 @@ class FolderAdvanced extends React.Component {
     }
   }
 
-  componentDidMount () {
-    this.loadContent()
+  async componentDidMount () {
+    await this.loadContent()
+    this.buildBreadcrumbs()
   }
 
   componentDidUpdate (prevProps, prevState) {
     const { state } = this
 
     if (prevState.content.content_id !== state.content.content_id) {
       this.loadContent()
+      this.buildBreadcrumbs()
```

Comment:
in componentDidMount, you added an await which you didn't add here. Is it intentional ?

Reply:
no, actually, await is needed here too to ensure the correctness of the breadcrumbs. I will make the change and resolve this conversation.